user/config-imports.md

```diff
@@ -0,0 +1,203 @@
+stored in your repository. You can import shared configuration snippets into
+your `.travis.yml` or [API build request payload](https://docs.travis-ci.com/user/triggering-builds/),
+so you can update your build configuration in multiple repositories making only
+one change.
+
+Imported configs can themselves include other configs, making this feature very
+composable (cyclic imports will be skipped). You can import up to 25 build
+configuration snippets in total.
+
+
+> BETA Configuration imports is currently in beta. Please leave feedback on the [Community forum](https://travis-ci.community/c/early-releases).
+{: .beta }
+
+## Example
+
+Instead of specifying which versions of Ruby to test against in multiple files
+across many repositories, you can define them in a shared snippet:
+
+```yaml
+rvm:
+  - 2.5
+  - 2.6
+```
+{: data-file="rubies.yml"}
+
+You can then `import` that snippet into your `.travis.yml`. The following
+configuration imports the file `rubies.yml` from the `main` branch of the
+`shared-configs` repository of the `travis-ci` account:
+
+```yaml
+import: travis-ci/shared-configs:rubies.yml@main
+script: bundle exec rake
+```
+{: data-file=".travis.yml"}
+
+Resulting in the following configuration:
+
+```yaml
+rvm:
+  - 2.5
+  - 2.6
+script: bundle exec rake
+```
+{: data-file=".travis.yml"}
+
+## Import sources
+
+The format of the `import` source is `<account>/<repository>:<path>@<ref>` in
+which `<ref>` can be any valid Git reference, such as a commit sha, branch name,
+or tag name.
+
+> Public repositories can import sources from public repositories, but not
+> private repositories. Private repositories can import sources from both
+> public and private repositories. See [private repositories](#private-repositories)
+> for more information.
+
+Import a single source:
+
+```yaml
+import: travis-ci/build-configs:rubies.yml@main
+
+# or
+
+import:
+  source: travis-ci/build-configs:rubies.yml@main
+```
+{: data-file=".travis.yml"}
+
+Import multiple sources:
+
+```yaml
+import:
+  - travis-ci/build-configs:rubies.yml@adf1235
+  - travis-ci/build-configs:other.yml@v1
+
+# or
+
+import:
+  - source: travis-ci/build-configs:rubies.yml@adf1235
+  - source: travis-ci/build-configs:other.yml@v1
+```
+{: data-file=".travis.yml"}
+
+By default, if the same keys are present in multiple imported snippets, they
+are overwritten with the last configuration imported. But you can also specify
+[deep_merge](#merge-mode).
+
+## Importing configs from the same repository
+
+When importing configurations stored in the same repository as your
+`travis.yml`, you can ommit the `<account>/<repository>` part:
+
+```yaml
+# local imports fetch the same git commit ref
+import:
+- one.yml
+- path/to/other.yml
+```
+{: data-file=".travis.yml"}
+
+The path is relative to the repository's root.
+
+## Importing specific versions of configs
+
+
+Configurations imported from a different repository, fetch the fetch the latest
```

Comment:
```suggestion
For configurations imported from a different repository, fetch the latest
```

Reply:
I'm not sure this has the same meaning. This has been Sam's (or Maria's?) wording. It is meant to say: 

> If you import a configuration from a different repository, the [version so-and-so] will be fetched/used/imported

Reply:
now made this:

> For configurations imported from a different repository the latest version of
the default branch in the repository will be used by default.

and

> For configurations imported from the same repository the commit you are
currently building will be used by default. This is intended to help while you
are creating and testing the shared configurations.